addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/type/HomematicChannelTypeProviderImpl.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.homematic.type;
```

Comment:
Is this necessary? I don't like to have any dependencies to openHab in the internal package.